src/ActionUnit.cpp

```diff
@@ -313,6 +313,7 @@ std::list<QPointer<TToolBar>> ActionUnit::getToolBarList()
                 constructToolbar(childAction, pTB);
                 childAction->mpToolBar = pTB;
                 pTB->setStyleSheet(pTB->mpTAction->css);
+                pTB->setStyleSheet(mpHost->mProfileStyleSheet);
```

Comment:
:stop_sign: This, and the other changes in this commit are going to cancel the effects of the user entering *anything* into the style-sheet entry box in the "Buttons" editor view (which gets stored into the `(QString) css` member of the `TAction` associated with each button/menu/toolbar instance).

Reply:
Just tested it again. It doesn't cancel the effects of the button style-sheet. Am I just not seeing something?

Reply:
Well if `pTB->mpTAction->css` does contain anything that styles `pTB` it will get wiped out by the `mpHost->mProfileStyleSheet` that is immediately follows it unless that also contains the same stuff that affects `pTB` - which then makes putting anything into the `Stylesheet` entry widget pointless:

![image](https://user-images.githubusercontent.com/6163092/75121437-de67aa00-568b-11ea-8e6e-0d96593df299.png)

Reply:
Even though I didn't happen for me, after thinking a bit. I think it's better to remove the possibility to change the Buttons and Buttontoolbar with setProfileStyleSheet because they already have their own ProfileStyleSheet in form of the mentioned css. 
Is there a option to change that css per script? If not that maybe would a nice future feature but with a new function in a new PR.